index.js

```diff
@@ -429,6 +429,8 @@ FSWatcher.prototype.unwatch = function(paths) {
   paths.forEach(function(path) {
     if (this._closers[path]) {
       this._closers[path]();
+      delete this._closers[path];
+      delete this._getWatchedDir(sysPath.dirname(path)).remove(sysPath.basename(path));
```

Comment:
Why is there a `delete` on this line? I think perhaps just `this._getWatchedDir(sysPath.dirname(path)).remove(sysPath.basename(path));` instead of both these lines would solve the issue without side effects.

(I suspect deleting the closer is a neutral change, not harmful, but also not necessary)

Haven't yet tried it out, but want to carve out some time today to do so.